dissect/target/plugins/os/windows/registry.py

```diff
@@ -329,7 +329,7 @@ def mappings(self) -> dict[str, str]:
     @internal
     def get_user_details(self, key: RegistryKey) -> UserDetails:
         """Return user details for the user who owns a registry hive that contains the provided key"""
-        if not key.hive or not key.hive.filepath:
+        if not key.hive or not hasattr(key.hive, "filepath") or not key.hive.filepath:
```

Comment:
```suggestion
        if not key.hive or not getattr(key.hive, "filepath", None):
```
We can simplify it a bit more 😄